### PR TITLE
upgrade post-css and autoprefixer

### PR DESCRIPTION
**Pull request by notlee**
fix component compilation error, due to use of `@supports selector` https://github.com/postcss/autoprefixer/issues/1391

### CODE REVIEW

**Review by akomiqaia — APPROVED** [review]

File: lib/middleware/v3/bundleSass.js [hunk 1]
```diff
@@ -38,18 +38,16 @@ async function bundleSass(location, entryPointPath, loadPaths=[]) {
...
-			'ie >= 11',
-			'ff ESR',
+			'Firefox ESR',
 			'safari >= 9'
 		],
 		cascade: false,
-		flexbox: 'no-2009',
-		grid: true
+		flexbox: false,
+		grid: false
```
Comment (notlee) [inline lib/middleware/v3/bundleSass.js:50]: Updated to reflect our browser support policy and the browsers we test with

File: lib/middleware/v3/bundleSass.js [hunk 2]
```diff
@@ -38,18 +38,16 @@ async function bundleSass(location, entryPointPath, loadPaths=[]) {
...
 	const css = await fs.readFile(path.join(location, 'output.css'), 'utf-8');
 
-
 	const prefixedCss = await postcss([autoprefixer({
 		overrideBrowserslist: [
 			'> 1%',
 			'last 2 versions',
-			'ie >= 11',
-			'ff ESR',
+			'Firefox ESR',
```
Comment (notlee) [inline lib/middleware/v3/bundleSass.js:45]: We no longer support IE

File: package-lock.json [hunk 3]
```diff
@@ -12987,54 +12357,6 @@
         "darwin"
       ]
     },
-    "node_modules/sass-bin-linux-32": {
```
Comment (notlee) [inline package-lock.json:12990]: whhhhhyyyyy

**Comment by origamiserviceuser** [comment]
:tada: This PR is included in version v366.4.0 :tada:

**Pull request merged**